packages/s3-store/index.ts

```diff
@@ -517,4 +522,31 @@ export class S3Store extends DataStore {
 
     this.saveMetadata(file, upload_id)
   }
+
+  public async remove(id: string): Promise<void> {
+    try {
+      const {upload_id} = await this.getMetadata(id)
+      if (upload_id) {
+        await this.client
+          .abortMultipartUpload({
+            Bucket: this.bucket,
+            Key: id,
+            UploadId: upload_id,
+          })
+          .promise()
+      }
+    } catch (error) {
+      log('remove: No file found.', error)
+      throw ERRORS.FILE_NOT_FOUND
```

Comment:
Wondering if it makes sense to look at the error, because not all errors mean the file is not found (meaning it's already deleted)?

We do something similar here: 

https://github.com/tus/tus-node-server/blob/95366fbac8ed692aaf1a54d9821fd4f7877fd54a/packages/s3-store/index.ts#L190-L199